README.md

```diff
@@ -45,7 +45,7 @@ pip3 install -r requirements.txt
 ```
 
 ## Preparing the configuration
-Create the `sbs-config.json` file and add the following content in the file:
+Create the `sbs-config.json` file in any location you choose, and add the following content in the file:
```

Comment:
perhaps, "in any location your choose on your local machine".

Reply:
Added this update. Thank you very much.